src/AprsISLibrary/TcpConnection.cs

```diff
@@ -0,0 +1,64 @@
+﻿ namespace AprsSharp.Connections.AprsIs
+{
+    using System;
+    using System.IO;
+    using System.Net.Sockets;
+
+    /// <summary>
+    /// Represents a TcpConnection.
+    /// </summary>
+    public sealed class TcpConnection : ITcpConnection, IDisposable
+    {
+        private readonly TcpClient tcpClient = new TcpClient();
+        private NetworkStream? stream;
+        private StreamWriter? writer;
+        private StreamReader? reader;
+
+        /// <inheritdoc/>
+        public void Connect(string server, int port)
+        {
+            tcpClient.Connect(@server, 14580);
```

Comment:
I think this should reference the port passed in by the caller, right?

```suggestion
            tcpClient.Connect(server, port);
```